workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/utils/api.ts

```diff
@@ -941,3 +941,41 @@ export async function updateProductSpecification({
 
 	return await response.json();
 }
+export async function updateUserAdditionalinfos(body: Object, id: number) {
+	const response = await fetch(
+		`${baseURL}/o/c/useradditionalinfos/${id}/?filter=contains(sendType,'shipping')`,
+		{
+			body: JSON.stringify(body),
+			headers,
+			method: 'PATCH',
+		}
+	);
+
+	return await response.json();
+}
+
+export async function getMyUserAditionalInfos(userId: number) {
+	const userAdditionalInfos = await fetch(
+		`${baseURL}/o/c/useradditionalinfos/?filter=r_userToUserAddInfo_userId eq '${userId}' and contains(sendType,'shipping')`,
+		{headers}
+	);
+
+	const response = await userAdditionalInfos.json();
+
+	if (!response.acceptInviteStatus) {
+		return response;
+	}
+}
```

Comment:
You have two API call functions with this name. Try to turn it more generic using just one for both cases.